File: BitManipulation/LeetCode/Easy/SingleNumber1/SingleNumber1.java
// https://leetcode.com/problems/single-number/
public class SingleNumber1 {
    public int singleNumber(int[] nums) {
        int xor = 0;
        for (int i : nums) {
            xor = xor ^ i;
        }
        return xor;
    }
}

File: BitManipulation/LeetCode/Easy/SingleNumber1/SingleNumber1.cpp
// https://leetcode.com/problems/single-number/
#include<bits/stdc++.h>
using namespace std;
int singleNumber(vector<int> &nums)
{
    int XOR = 0;
    for (int i : nums)
    {
        XOR = i ^ XOR;
    }
    return XOR;
}
int main(void) {
    
    return 0;
}